README.md

```diff
@@ -1,6 +1,77 @@
 # LLM Compressor
+Easy to use model compression library that supports a growing list of formats including
+int and float quantization, activation quantization, and 2:4 sparsity.
 
-A library for compressing large language models utilizing the latest techniques and research in the field for both training aware and post training techniques. 
-The library is designed to be flexible and easy to use on top of PyTorch and HuggingFace Transformers, allowing for quick experimentation.
+## Overview
+A library for compressing large language models utilizing the latest techniques and research for both training aware and post training techniques.
+llm-compressor is designed to be flexible and easy to use on top of PyTorch and HuggingFace Transformers, allowing for quick experimentation.
+Compression algorithms are implemented as `Modifiers` which can be applied to create optimized models for inference.
+The library also emphasises support for deployment on vLLM through export to the compressed-tensors format.
 
-## More Coming Soon!
\ No newline at end of file
+## Installation
+
+### Pip
+
+Coming Soon!
+
+### From Source
+llm-compressor can be installed from the source code via a git clone and local pip install.
+
+```bash
+git clone https://github.com/vllm-project/llm-compressor.git
+pip install -e llm-compressor
+```
+
+## Quick Tour
+The following snippet is a minimal example for compression and inference of a Llama model.
+The 1.1B model may be swapped to another model in the HuggingFace Hub or custom local model.
+This example uses 4-bit weight quantization, however the `scheme` may be changed to
+target different quantization scenarios.
+
+
+### Compression
+Compression is easily applied by selecting an algorithm (GPTQ) and calling the `oneshot` API.
+
+```python
+from llmcompressor.transformers import oneshot
+from llmcompressor.modifiers.quantization.gptq import GPTQModifier
+
+# sets parameters for the GPTQ algorithms - target Linear layer weights at 4 bits
+gptq = GPTQModifier(scheme="W4A16", targets="Linear")
+
+oneshot(
+    model="TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T",  # sample model
```

Comment:
I would update this to use `TinyLlama/TinyLlama-1.1B-Chat-v1.0`

Reply:
I think we should instead pass a dataset that the user created here, like this example

- https://github.com/robertgshaw2-neuralmagic/vllm-examples/blob/main/quantization/w8a8-example.py

The "open_platypus" is a bit too opaque

Reply:
agreed - wanted to keep the code minimal in the readme, taking a look at what we can do